benchmarks/fogbench/__main__.py

```diff
@@ -193,13 +241,18 @@ def check_coap_server():
 parser.description = '%(prog)s -- a Python script used to test FogLAMP (simulate payloads)'
 parser.epilog = 'The initial version of %(prog)s is meant to test the sensor/device interface of FogLAMP using CoAP'
 parser.add_argument('-v', '--version', action='version', version='%(prog)s {0!s}'.format(_FOGBENCH_VERSION))
-parser.add_argument('-k', '--keep', default=False, choices=['y', 'yes', 'n', 'no'], help='Do not delete the running sample (default: no)')
+parser.add_argument('-k', '--keep', default=False, choices=['y', 'yes', 'n', 'no'],
+                    help='Do not delete the running sample (default: no)')
 parser.add_argument('-o', '--output', help='set the output file, WITHOUT extension')
 
 parser.add_argument('-I', '--iterations', help='The number of iterations of the test (default: 1)')
 parser.add_argument('-O', '--occurrences', help='The number of occurrences of the template (default: 1)')
 
 parser.add_argument('-H', '--host', help='CoAP server host address (default: localhost)', action=check_coap_server())
+parser.add_argument('-i', '--interval', default=0, help='The interval in seconds for each iteration (default: 0)')
+
+parser.add_argument('-S', '--statistics', default='total', choices=['total', 'st', 'et', 'mt', 'bt',
+                                                                    'itr', 'mt_itr', 'bt_itr'], help='The type of statistics to collect (default: total)')
```

Comment:
I suggest to make these 'mt_itr', 'bt_itr' hyphen `-` separated, instead of underscore `_`